app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/UploadSupportingEvidenceAnswers.scala

```diff
@@ -34,12 +37,13 @@ object UploadSupportingEvidenceAnswers {
 
   final case class IncompleteUploadSupportingEvidenceAnswers(
```

Comment:
Let's rename these:

`UploadSupportingEvidenceAnswers` -`SupportingEvidenceAnswers`

Rip out the Upload prefix. It's all about supporting evidence and not about action of uploading.